src/plone/volto/behaviors/kicker.py

```diff
@@ -7,13 +7,17 @@
 
 
 @provider(IFormFieldProvider)
-class IHeadTitle(model.Schema):
+class IKicker(model.Schema):
 
-    head_title = schema.TextLine(
-        title=_("label_head_title", default="Header title"),
+    kicker = schema.TextLine(
```

Comment:
If we change the name of the field, it won't be backward-compatible.

There are a bunch of places [in volto](https://github.com/search?q=repo%3Aplone%2Fvolto%20head_title&type=code) and [in volto-light-theme](https://github.com/search?q=repo%3Akitconcept%2Fvolto-light-theme%20head_title&type=code) which assume there is a field called head_title, not kicker.

We also have [a catalog column](https://github.com/plone/plone.volto/blob/main/src/plone/volto/profiles/default/catalog.xml#L4) which copies the value of the head_title field.

So, it'll be a lot easier if we keep the existing internal field name and only change the name of the behavior and the user-visible text.

```suggestion
    head_title = schema.TextLine(
```

Reply:
@davisagli didn't we agree to add a new "kicker" field in addition to the head_title field and then deprecate head_title? Of course we would have to write a migration for existing projects at some point. However, we would have a clean codebase and not an inconsistency in our code base.

Reply:
@tisto No, I thought the plan was to add a new behavior but keep the old field name. If we change the field name then any component which displays the kicker needs to check both the old and new field name, so it would make a lot of code more messy (and also take some time to find and update each location).

We can talk about it tomorrow but I don't think it's worth the effort to change the field name.

Reply:
@davisagli ok, then this was a misunderstanding from my side. I leave this decision up to you. Just wanted to mention what I understood from our discussion.

Reply:
Just one last thing. I was under the assumption that we only support "kicker" in future releases. Since head_title never made it into core this would be a breaking change that is acceptable IMO.

Reply:
@tisto There are already references to `head_title` in core: https://github.com/search?q=repo%3Aplone%2Fvolto%20head_title&type=code

Reply:
done